clang/include/clang/Driver/Options.td

```diff
@@ -1554,7 +1554,10 @@ def fstrict_flex_arrays_EQ : Joined<["-"], "fstrict-flex-arrays=">, Group<f_Grou
   Visibility<[ClangOption, CC1Option]>,
   NormalizedValuesScope<"LangOptions::StrictFlexArraysLevelKind">,
   NormalizedValues<["Default", "OneZeroOrIncomplete", "ZeroOrIncomplete", "IncompleteOnly"]>,
-  HelpText<"Enable optimizations based on the strict definition of flexible arrays">,
+  HelpText<"Enable optimizations based on the strict definition of flexible arrays. "
```

Comment:
This is much too long to include in the `--help` output. Maybe this could be specified as `DocBrief` instead soon it goes into the command-line reference but not `--help`

Reply:
Done. I wasn't aware of DocBreef, that's indeed much better like this.